### ISSUE TITLE
Some URLs (parts) in Zalizniak-2020-2590 seem to be incorrect

### ISSUE THREAD

**Issue by xrotwang** [concepticon/concepticon-data#1217]
Triggered by my investigation of the URLs in the Bulakh list (see #1216), I looked at the URLs in Zalizniak-2020-2590. The first one I looked up was [shift328](https://datsemshift.ru/shift328), given for [concept `taste (n.)`](https://github.com/concepticon/concepticon-data/blob/master/concepticondata/conceptlists/Zalizniak-2020-2590.tsv#L1359), which seems to be an error. I checked a couple more, and these seemed to be fine.

So, I guess, what's to be done here is
- investigating whether the IDs in the SemShifts Database are supposed to be stable
- if that's the case, recheck the IDs we have extracted into our concept list.

**Comment by xrotwang** [comment]
We may also want to change the data in this list to a more meaningful format, e.g. modeling semantic shifts as JSON array of objects.

**Comment by LinguList** [comment]
How awful, I just saw that the datatype was somehow messed up: 
https://concepticon.clld.org/values/Zalizniak-2020-2590-1

The idea was to model this (as other network-like datatypes with 
relations) along these lines: https://calc.hypotheses.org/2617

I am open to JSON, but we'd need to update some of the tutorials then 
(or write a new one) and specifically these usecases, showing how I get 
a graph out of a dataset like the one by Zalizniak or MultiSimlex 
(Vulic) would be important then.

**Comment by LinguList** [comment]
I now see: the List-representation is following the fact that I 
specified the data format in CSVW. It is not an error on the side of the 
original data, which follows the format I describe in the blog post.

**Comment by xrotwang** [comment]
Ah, good point. I'll have an eye on this blog post when proposing a new data model. We'd have to rewrite such material anyway, considering the split between Concepticon and NoRaRe.

**Comment by LinguList** [comment]
Yes. In fact, given that this was some work that I did in a fashion you could consider "without peer review", as I just set it up and proposed it without alternatives, it is time to get this peer reviewed ;)

**Comment by LinguList** [comment]
And accordingly enhanced.

**Comment by xrotwang** [comment]
So, I'd say the data in Concepticon can stay as it is (and hence the blog post stays valid), but a NoRaRe extract of the Zalizniak data would have a JSON representation making some of the aggregation explained in the blog post unneccessary.

**Comment by xrotwang** [comment]
@LinguList what do you think about errors in the data? Could these be systematic? Is it worth fixing?

**Comment by xrotwang** [comment]
@LinguList @AnnikaTjuka and shouldn't the graph of shifts be added to NoRaRe, too?

**Comment by LinguList** [comment]
That would of course be beautiful. Then we could add this graph, the Bulakh graph, the CLICS graphs, etc. as well!

**Comment by LinguList** [comment]
We could zip (to save storage place) and use JSON or GML format.

**Comment by LinguList** [comment]
As to the errors: I am afraid they are unsystematic, I don't know what drives their decisions.

**Comment by LinguList** [comment]
![screenshot](https://user-images.githubusercontent.com/2916210/184524491-327715a8-e433-42c3-a232-f6c161adb14f.png)

This is how shift 328 looked in December 2020.

**Comment by LinguList** [comment]
[Wayback Machine](https://web.archive.org/web/20201103080404/http://datsemshift.ru/browse) confirms this.

**Comment by LinguList** [comment]
As to the format: GML -- though differently interpreted currently -- has the advantage of working with Cytoscape, one of the more important visualization tools for networks (which exports to stable HTML with d3, as well).

**Comment by xrotwang** [comment]
To construct the full graph from the tabular data, we'd need some assembly anyway, so I'd vote for simple JSON columns in the tabular data, plus a custom cldfbench command (in norare-cldf) which assembles GML graphs for the relevant datasets.

**Comment by chrzyki** [comment]
I've been periodically trying to automatically verify the links and investigate the URL issue, but DatSemShift has consistently been giving me HTTP 504 in the last couple of days. Should this be postponed until later?

The other question regarding JSON columns is more or less settled, right?

**Comment by xrotwang** [comment]
URLs have been removed from the 2020 list.
